docs/guide/http_cache.md

```diff
@@ -284,23 +293,23 @@ ezplatform:
     FASTLY_KEY="token"
     ```
 
-    ##### Configuration Fastly on Platform.sh
+    ##### Configuring Fastly on Platform.sh
```

Comment:
Is there a way we can deal with link breakage on things like this? (and other header changes)

Reply:
Only on page level, not on heading level AFAIK

Reply:
ok